.github/workflows/generate-event-pdfs.yml

```
@@ -2,7 +2,18 @@ name: generate-pdf-artifacts
 
 on:
   workflow_dispatch:
-    inputs: {}
+    inputs:
+      REMOVE_MASTER:
+        description: Remove the master files
```

I'd also want to suggest having this be more explicit / descriptive to what is being done - "Remove the master files" isn't a good indicator. I haven't read the rest of the PR but I'd guess we would want something along the lines of:

```suggestion
        description: If the environment variable REMOVE_MASTER is to to 1, remove all markdown events from the master branch.
```
Why would we want to do this? Feel free to edit the suggestion above.

---

```suggestion
        description: Remove unchanged files from master branch to generate only updated pdfs
```

see https://github.com/SORSE/sorse.github.io/pull/251#issuecomment-663910085